person.rb

```diff
@@ -1,15 +1,17 @@
 require_relative 'corrector'
+require_relative 'rental'
 
 class Person
   attr_reader :id, :corrector
-  attr_accessor :name, :age, :parent_permission
+  attr_accessor :name, :age, :parent_permission, :rentals
 
-  def initialize(age, name = 'unknown', parent_permission: true)
+  def initialize(age, name = 'unknown', rentals = [], parent_permission: true)
     @id = rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
     @corrector = Corrector.new
+    @rentals = rentals
```

Comment:
- [ ] **[OPTIONAL]** You probably do not need to define the **`rentals`** parameter here. The `rentals` variable can also be implicitly initialized as demonstrated in the suggestion above.